data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			`	You ask her if she's worried you've been compromised - that you might be working against the Skadenga's best interests. Hjlod shakes her head. "No. Future is set. Vhat vill, vill be. You are given task to tell us vhere to go, ve go vhere you say. The rest is just noise.`
+			label waiting
+			`	"Now", Hjlod says, "Tell me about Setstokkr. Did you find it?"`
+			choice
+				`	"I found nothing definitive."`
+					goto nothing
+				`	"I think I found traces of it on Windblain."`
+					goto nothing
+				`	"I found a strange box similar to the Setstokkr on Asgard."`
+			`	"Sounds like herbalist box. Ondurdis had one long before I vas born. Gave it to liar. It not important though, not Setstokkr."`
+			label nothing
+			`	"Sending big piece of vood through space probably not best idea Ondurdis had." Hjlod smirks. "Still, it does not matter so much what you find. Only how you felt. You saw the vorlds. Vhere should ve go?"`
+			choice
+				`	"Windblain."`
+				`	"Asgard."`
+					goto asgard
+			apply
+				set "Nifel"
+			`	You answer, "There is a nearly abandoned village called Hrithfjall next to a coastal God-Stone. Something about that place felt right. I believe that is where the Skadenga should migrate."`
+			`	Hjlod nods. "Hrithfjall sounds like good place. I vill tell Ondurdis.`
+			goto end
+			label asgard
+			apply
+				set "Asgard"
+			`	You answer, "On Asgard, there is a great market built around their God-Stone, and many new homes in need of people to fill them. The people there have lost their way and they need the Skadenga to remind them of who they were."`
+			`	Hjlod nods. "The Skadenga have alvays been guides, pathfinders. I vill tell Ondurdis.`
+			label end
+			`	"Vhen you were gone, thing, or assembly, vas held. Ondurdis called in all Skadenga from all villages across Nifel. You must find transportation for 2,200 folk. That is what Onderduis told me to tell you."`
+			`	You frown and ask, "Did she provide money?"`
+			`	Hjlod shakes her head. "No. Ve could not afford it. You must go to Valhalla and convince head of Transport Union to support us." She squeezes your shoulder. "Thank you for help. For finding us new home. Ve need it. I must report now to Ondurdis. Skade be vith you." She turns and leaves.`
+				accept
+	on complete
+		conversation
+			`	Outside your ship, you are met by a representative of the Transport Union. "We have been expecting you, Captain <last>." They escort you to an elevator and up to the penthouse of the tallest building in Valhalla's capital of Himinbjorg."`
+			scene "scene/tower"
+			`	Three youngish men adorned in trending clothes smile as you enter. "Ah, Captain <last>, what a pleasure to finally meet you. I am Kaiden, and these are my associates Braiden and Grayden. We are executives of the Transport Union. Obviously, we were made aware you would be coming and are looking forward toward creating a lasting and beneficial relationship with you and your... clients.`
```

Comment:
These characters feels too... "on the nose" bad/evil to me in a way that doesn't blend well with the rest of ES. The way I see most of the game, the vast majority of characters (maybe excluding alphas) are at conflict not because there's good guys and bad guys, but just people who have different points of view. I don't get that same feeling here. I doubt anyone would think to themselves, "I agree with these people," or even, "I get where they're coming from." The thing that especially stands out is the derogatory tone/words used toward the Skadenga (`rustic primitives`, `savages`, etc). I'd suggest some rewriting that maybe tones down the whole "17th century colonists vs. natives" vibe.

Reply:
As far fetched as it seems, I would argue that decisions/thinking like those can be found in powerful corporate executive circles. There are poor people, native people, and just regular people today who suffer because of short-sighted / profit motived decisions. It's not a 17th century problem, it's a today problem, and will be a tomorrow problem. 

I can remove some of derogatory references if you really think its over top, but I think you're giving humanity way too much credit if think it's unbelievable.

Reply:
I gotcha. I still do get the vibe that it's over the top for me - I think a good compromise would be removing some of the derogatory references, as you suggested. The ones I'd specifically suggest rewording/taking out are uses of `sad savages` and `misfit primitives`. There are a couple instances too outside of this specific conversation with Carla, the narration, and others using them - consider rewording those too.

Reply:
Alright. I removed some instances of 'primitive' and 'savage' from the text. This includes the instances you brought up, plus a few others.

Reply:
Some of the changes I like, some are still "eh" to me, personally. I was more hoping you'd reword into something less aggressive rather than at the same level of hostility. I'll throw in a few suggestions myself and see if you like them.

Reply:
I have accepted your suggestions.